tests/unit/jobs_workers/public_jobs_workers/fix_nested_jobs/test_fix_nested_jobs_processor.py

```
@@ -53,49 +55,49 @@ def mock_services(monkeypatch: pytest.MonkeyPatch):
 class TestFixNestedJobsProcessorSteps:
     def test_verify_step_success(self, mock_services) -> None:
         processor = _make_processor()
-        processor.result["stages"]["fix"]["status"] = "success"
-        processor.result["file_result"] = {"path": "/tmp/test.svg", "nested_tags_before": 2}
-        mock_services["verify_fix"].return_value = {"after": 0, "fixed": 2}
+        processor.result.stages.fix.status = "success"
+        processor.result.file_result = FileResult(path="/tmp/test.svg", nested_tags_before=2)
```

Ruff (0.15.15)

[error] Probable insecure usage of temporary file or directory: "/tmp/test.svg"

(S108)